### PR TITLE
Move `/bank` page to v3

### PR DESCRIPTION
**Pull request by eilla1**
This PR moves the `/bank` page from v2 to v3 (Next.js + theme-ui). Sorry for taking forever on this PR 😅 

to-do before it's ready to review:
- [x] Landing Section
- [x] Features Section 
- [x] Events Section (`Testimonials.js`) 
- [x] "Everything you'll need" Section
- [x] Sign Up section (`Start.js`)

### CODE REVIEW

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey Ella! Thanks for the wonderful work on this PR, a few styles notes from me at the moment:


image


image

image

image

image
image

image


image

**Comment by sampoder** [comment]
Apologies for the poor handwringing, it may not leak through in my annotations as they are negative lol but thank you so so much for doing this, i really appreciate it and this is an incredible job you've down. Hats off to you

**Comment by sampoder** [comment]
And the poor spelling.... everywhere

**Comment by eilla1** [comment]
Thanks for the review Sam! Really appreciate the detailed feedback!

i just have one more thing to fix hehe

**Comment by eilla1** [comment]
Screen Shot 2021-08-16 at 9 37 31 AM

The timeline looks like this for me 🤔

File: components/bank/Timeline.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+function Circle({ children }) {
+  return (
+    <Box
+      style={{
+        padding: 12,
+        background: 'red',
+        color: 'white',
+        backgroundImage:
+          'radial-gradient(ellipse farthest-corner at top left, #ff8c37, #ec3750',
```
Comment (sampoder) [inline components/bank/Timeline.js:58]:
```suggestion
          'radial-gradient(ellipse farthest-corner at top left, #ff8c37, #ec3750)',
```

Comment (sampoder) [reply components/bank/Timeline.js:58]: This should fix the timeline CSS

**Review by sampoder — CHANGES_REQUESTED** [review]
A few more notes on styles, thanks as always Ella

IMG_0943
IMG_0942
IMG_0941
IMG_0940
IMG_0939
IMG_0938

**Review by sampoder — CHANGES_REQUESTED** [review]
Could we balance out the vertical padding between the paragraph and the heading and the image, so they're all equal:

Screenshot 2021-08-20 at 11 49 26 AM

Could we fix the events on mobile:

Screenshot 2021-08-20 at 11 50 27 AM

The seven is also broken on mobile:

Screenshot 2021-08-20 at 11 51 00 AM

Reduce the vertical padding here on mobile:

Screenshot 2021-08-20 at 11 51 20 AM

Have the timeline centered on mobile and the lines end at the circles and not go through 

Screenshot 2021-08-20 at 11 52 01 AM

This text looks weird on mobile:

Screenshot 2021-08-20 at 11 53 08 AM

Thanks Ella!

**Review by sampoder — CHANGES_REQUESTED** [review]
This is really starting come together! The finish light is in sight, thank you so so much Ella!

Couple of notes, almost solely on mobile formatting (hooray! /s),

the Hack Club flag covers our header on the smallest of screens, could we add some more vertical padding to ensure this is avoided? 

Screenshot 2021-08-21 at 5 37 28 PM

These lines aren't meeting the nodes :sadge:, if this is too much of a pain, maybe we remove the line on mobile?

Screenshot 2021-08-21 at 5 52 35 PM

These don't all line up:

Screenshot 2021-08-21 at 5 54 07 PM

There's still some horizontal overflow on the event cards:

Screenshot 2021-08-21 at 5 55 16 PM

Could we remove the left padding on the attendees stat so these all line up?

Screenshot 2021-08-21 at 5 55 49 PM

Could we reduce the gap between the image and the text content here?

Screenshot 2021-08-21 at 5 56 39 PM

Once again, thank you so so much Ella, I can't say it enough

**Comment by eilla1** [comment]
Appreciate the thorough review, thanks Sam :D

Along with the other style issues, I also fixed the lines on the timeline to be a fixed height instead of a percentage, so now it should work on all screen heights

**Review by sampoder — APPROVED** [review]
Hoorah! I think this is ready to ship! thank you so much Ella for all of this, this is amazing!

**Pull request merged**